docs/src/main/asciidoc/telemetry-micrometer-tutorial.adoc

```diff
@@ -3,8 +3,9 @@ This guide is maintained in the main Quarkus repository
 and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
```

Comment:
Do we have a redirect in place for the `micrometer.adoc` file you removed?

Reply:
I made it.. did I submit it is the question.

Let me retrace my steps yesterday to see when I got distracted and dropped the thread...

Reply:
https://github.com/quarkusio/quarkusio.github.io/pull/1709